.changeset/breezy-experts-complain.md

```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-query': patch
+---
+
+Add enumeration support for aggregation projection columns.
```

Comment:
Let's add a separate changeset for the fix of #1079 as we usually do

Reply:
why a separate one??